e2e/tests/oauth/test_client_credentials.py

```diff
@@ -141,10 +141,10 @@ def test_incorrect_jwt_algorithm(
                 {"iss"},
             ),
             (
-                # Test invalid jti
+                # Test invalid jti - integer
                 {
                     "error": "invalid_request",
-                    "error_description": "Failed to decode JWT",
+                    "error_description": "Jti claim must be a unique string such as UUID4",
```

Comment:
I've made up this error message - assume it will need improving by someone else?

Reply:
check it with Aubyn...